### PR TITLE
Check and update errors in drive event handling

### PR DESCRIPTION
**Pull request by balamurugana**

### CODE REVIEW

File: pkg/drive/event.go [hunk 1]
```diff
@@ -345,10 +350,62 @@ func (handler *driveEventHandler) handleUpdate(ctx context.Context, drive *types
...
+				klog.ErrorS(uerr, "unable to set I/O error", "drive", drive.GetDriveID())
+			}
+		} else {
+			klog.ErrorS(err, "unable to check volume root directory existent", "drive", drive.GetDriveID())
+		}
+
+		return nil
+	}
+
+	if _, err := handler.getDeviceByFSUUID(drive.Status.FSUUID); err != nil {
```
Comment (Praveenrajmani) [inline pkg/drive/event.go:370]: we should also revert the state back to "Ready" when the device by FSUUID is found

Comment (balamurugana) [reply pkg/drive/event.go]: done

File: pkg/drive/event.go [hunk 2]
```diff
@@ -345,10 +350,92 @@ func (handler *driveEventHandler) handleUpdate(ctx context.Context, drive *types
...
+				"unable to find device by FSUUID; "+
+					"either device is removed or run command "+
+					"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
+					"on the host to reload",
+				"FSUUID", drive.Status.FSUUID)
+			client.Eventf(
+				drive, client.EventTypeWarning, client.EventReasonDeviceNotFoundError,
+				"unable to find device by FSUUID %v; "+
+					"either device is removed or run command "+
+					"`sudo udevadm control --reload-rules && sudo udevadm trigger`"+
```
Comment (Praveenrajmani) [inline pkg/drive/event.go:380]: actually, `udevadm trigger` will not fix the symlink problems sometime. we may require a reboot in such cases. do we need to communicate this information as well?

Comment (balamurugana) [reply pkg/drive/event.go:380]: No. It is not a general case. `udevadm trigger` is general guideline whereas node reboot is specific case.

**Review by Praveenrajmani — APPROVED** [review]

**Pull request merged**